scripts/push_docs_to_branch.sh

```diff
@@ -31,10 +31,12 @@ NEW_VERSION=
 EXISTING_VERSION=
 if [[ "$GITHUB_REF" =~ $PATTERN_FOR_NEW_VERSION ]]
 then
-    NEW_VERSION=${GITHUB_REF/refs\/tags\//}
-elif [[ "$GITHUB_REF" =~ $PATTERN_FOR_MICRO_VERSION ]]
+    NEW_VERSION=$(echo $GITHUB_REF | sed -E "s/^refs\/tags\/([0-9]+)\.([0-9]+)\.0$/\1.x/")
+    if [[ -n ${CI} ]]; then echo "New version: ${NEW_VERSION}"; fi
+elif [[ "$GITHUB_REF" =~ $PATTERN_FOR_EXISTING_VERSION ]]
 then
-    EXISTING_VERSION=$(echo $GITHUB_REF | sed -E "s/^refs\/tags\/([0-9]+)\.([0-9]+)\.[0-9]+$/\1.\2.0/")
+    EXISTING_VERSION=$(echo $GITHUB_REF | sed -E "s/^refs\/tags\/([0-9]+)\.([0-9]+)\.[0-9]+$/\1.x/")
+    if [[ -n ${CI} ]]; then echo "Existing version: ${EXISTING_VERSION}"; fi
 fi
 
```

Comment:
at line 50 can you add lines similar to [these one](https://github.com/RasaHQ/rasa/blob/538849e0eef0044c9f62799ce944cc54ad2d2c36/scripts/push_docs_to_branch.sh#L53-L54)? I think this is the reason why `fs-extra` is not installed (as you can seem, the lock file didn't change, meaning that the dependency was already installed)